WalletWasabi/WabiSabi/Client/Batching/InProgressPayment.cs

```diff
@@ -0,0 +1,17 @@
+using NBitcoin;
+
+namespace WalletWasabi.WabiSabi.Client.Batching;
+
+// Represents a payment that has being chosen for a coinjoin round. 
+public record InProgressPayment(IDestination Destination, Money Amount, uint256 RoundId) 
```

Comment:
It is there because I thought it was exactly as you described but then I realized there can be only one round so, `RoundId` is not necessary but then I considered it would be okay to let it there just in case. But yes, basically it should be removed if it is not used.